PWGEM/PhotonMeson/Core/EMEventCut.h

```diff
@@ -61,17 +61,17 @@ class EMEventCut : public TNamed
   {
     switch (cut) {
       case EMEventCuts::kFT0AND:
-        return collision.sel8();
+        return (!mRequireFT0AND || collision.sel8());
```

Comment:
can you move mRequireFT0AND to line 44?
 if (mRequireFT0 && !IsSelected(collision, EMEventCuts::kFT0AND)) {
 same for requireNoTFB and requireNoITSROFB.

Reply:
Yes, that seems better. I implemented it in the update